src/HealthGPS/simulation.cpp

```diff
@@ -113,27 +96,27 @@ adevs::Time HealthGPS::update(adevs::SimEnv<int> *env) {
     return adevs_inf<adevs::Time>();
 }
 
-adevs::Time HealthGPS::update(adevs::SimEnv<int> * /*env*/, std::vector<int> & /*x*/) {
+adevs::Time Simulation::update(adevs::SimEnv<int> * /*env*/, std::vector<int> & /*x*/) {
     // This method is never called because nobody sends messages.
     return adevs_inf<adevs::Time>();
 }
 
-void HealthGPS::fini(adevs::Time clock) {
+void Simulation::fini(adevs::Time clock) {
```

Comment:
Is it just me or is `fini` a weird name for this method?

And if it's only ever called at the end of a simulation, maybe we should figure out how to do this using a destructor.

Reply:
It's required/called by the strange ADEVS library. TBH we can just do away with it in future work. I can't see any benefit it introduces over making our own simple runner.

Reply:
Oh, yuck. Let's totally put that on the chopping board too